test/mcp-2026-07-28/server_2026.py

```
@@ -0,0 +1,159 @@
+        await asyncio.sleep(float(os.environ.get("MCP_RIG_STREAM_GAP", "0.4")))
+        payload = json.dumps({"jsonrpc": "2.0", "method": "notifications/message",
+                              "params": {"seq": index}}).encode()
+        await send({"type": "http.response.body", "body": b"data: " + payload + b"\n\n",
+                    "more_body": True})
+    await send({"type": "http.response.body", "body": b"", "more_body": False})
+
+
+def with_stream_route(app):
+    """Route /stream to the long-lived responder, everything else to MCP."""
+
+    async def wrapped(scope, receive, send):
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `wrapped`

Add return type annotation: `None`

(ANN202)